test/test-functions

```diff
@@ -940,6 +940,8 @@ install_fs_tools() {
     # we use mkfs in system-repart tests
     image_install /sbin/mkfs.ext4
     image_install /sbin/mkfs.vfat
+    image_install /sbin/mkfs.btrfs
```

Comment:
This needs to be installed via `image_install -o` preferably in `test_append_files()` in `test/TEST-58-REPART/test.sh` and the respective test needs to be guarded by condition, since btrfs might not be available everywhere (e.g. CentOS).